packages/node-experimental/src/sdk/client.ts

```diff
@@ -17,6 +26,54 @@ class NodeExperimentalBaseClient extends NodeClient {
 
     super(options);
   }
+
+  /** Get the OTEL tracer. */
+  public get tracer(): Tracer {
+    if (this._tracer) {
+      return this._tracer;
+    }
+
+    const name = '@sentry/node-experimental';
+    const version = SDK_VERSION;
+    const tracer = trace.getTracer(name, version);
+    this._tracer = tracer;
+
+    return tracer;
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public async flush(timeout?: number): Promise<boolean> {
+    const provider = this.traceProvider;
+    const spanProcessor = provider?.activeSpanProcessor;
+
+    if (spanProcessor) {
+      await spanProcessor.forceFlush();
+    }
+
+    return super.flush(timeout);
+  }
+
+  /**
+   * Extends the base `_prepareEvent` so that we can properly handle `captureContext`.
+   * This uses `new Scope()`, which we need to replace with our own Scope  for this client.
+   */
+  protected _prepareEvent(event: Event, hint: EventHint, scope?: Scope): PromiseLike<Event | null> {
+    let actualScope = scope;
+
+    // Remove `captureContext` hint and instead clone already here
+    if (hint && hint.captureContext) {
+      actualScope = getFinalScope(scope, hint.captureContext);
+      delete hint.captureContext;
+    }
+
+    return super._prepareEvent(event, hint, actualScope);
+  }
 }
 
-export const NodeExperimentalClient = wrapClientClass(NodeExperimentalBaseClient);
+function getFinalScope(scope: Scope | undefined, captureContext: CaptureContext): Scope | undefined {
```

Comment:
l: We should explain what "finalScope" means.

Reply:
yeah, I kind of copied the naming from prepareEvent 😅 will elaborate a bit.